pkg/logcli/query/query.go

```diff
@@ -420,6 +420,27 @@ func (q *Query) printResult(value loghttp.ResultValue, out output.LogOutput, las
 	return length, entry
 }
 
+// loadRemoteSchema First tries to download the tenant specific schema config (tenant-schemaconfig.yaml),
+// if it doesn't exist, fallback to the global one (schemaconfig.yaml) for backwards compatibility.
+func loadRemoteSchema(client chunk.ObjectClient, tenant string) (*config.SchemaConfig, error) {
+	tenantSchemaConfigFilename := fmt.Sprintf("%s-%s.yaml", tenant, schemaConfigFilename)
+	globalSchemaConfigFilename := fmt.Sprintf("%s.yaml", schemaConfigFilename)
+	loadedSchema, err := LoadSchemaUsingObjectClient(client, tenantSchemaConfigFilename)
+	if err != nil {
+		if !strings.Contains(err.Error(), "object doesn't exist") {
+			return nil, fmt.Errorf("failed to load schema config: %w", err)
+		}
+
+		log.Printf("failed to load tenant specific schema config (%s). Falling back to load global schema", tenantSchemaConfigFilename)
```

Comment:
```suggestion
		log.Printf("failed to load tenant specific schema config (%s). Falling back to the global schema", tenantSchemaConfigFilename)
```
It's probably better to print the global schema config file as well.